service/controller/resource/blobobject/desired.go

```diff
@@ -25,11 +27,78 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	var clusterCerts certs.Cluster
+	var masterCertFiles []certs.File
+	var workerCertFiles []certs.File
```

Comment:
We do this twice in different handlers. That could be refactored but I was lazy now. 😬

Reply:
Since this is `blobobject`, this will most probably go away at some point. All good :+1: